packages/cli/lib/base-generator.js

```diff
@@ -83,15 +92,24 @@ module.exports = class BaseGenerator extends Generator {
   async setOptions() {
     let opts = {};
     const jsonFileOrValue = this.options.config;
+    debug(
+      'Loading generator options from CLI args and/or stdin.',
+      ...(this.option.config === undefined
+        ? ['(No config was provided.)']
+        : ['Config:', this.options.config]),
```

Comment:
I hope this is not too cryptic. 

When there is no config provided, I want to print:

`(No config was provided.)`

When there is some config provided, I want the value to be printed as a formatted object, prefixed with `Config: ` message, e.g.

`Config: {foo: 'bar'}`